integrationtests/model_storage/integrationtest_model_storage.py

```diff
@@ -96,13 +98,13 @@ def insert_triggers_into_database(
             full_model_interval,
         )
 
-        trigger_parent = Trigger(trigger_id=0, pipeline_id=pipeline_id)
-        trigger_child = Trigger(trigger_id=1, pipeline_id=pipeline_id)
+        trigger_parent = Trigger(trigger_id=parrent_trigger_id, pipeline_id=pipeline_id)
+        trigger_child = Trigger(trigger_id=child_trigger_id, pipeline_id=pipeline_id)
         database.session.add(trigger_parent)
         database.session.add(trigger_child)
         database.session.commit()
 
-        return pipeline_id, trigger_parent.trigger_id, trigger_child.trigger_id
```

Comment:
return statement shouldn't be in the `with` context because otherwise `__exit__()` is not executed.

Reply:
Ah I see thanks. but this is not a good practice anyway